aedes.js

```diff
@@ -257,23 +257,36 @@ Aedes.prototype.registerClient = function (client) {
 }
 
 Aedes.prototype._finishRegisterClient = function (client) {
+  var that = this
   this.connectedClients++
   this.clients[client.id] = client
   this.emit('client', client)
   this.publish({
     topic: '$SYS/' + this.id + '/new/clients',
     payload: new Buffer(client.id, 'utf8')
   }, noop)
+
+  this.publish({
+  topic: '$SYS/' + this.id + '/clients/total',
```

Comment:
I think it should be $SYS/broker/clients/total and it should be a retained message